support-frontend/app/services/MParticleClient.scala

```diff
@@ -0,0 +1,188 @@
+package services
```

Comment:
if you look at other things, e.g. 
https://github.com/guardian/support-frontend/blob/ac3193c39123b493358ca1098acf990a2d674e2c/support-frontend/app/services/ProductCatalogService.scala
you can see how much of the boilerplate is in the web service helper.  You can also see how it refreshes regularly, although that approach might not match what you need.
Have a look through, especially the more recently added ones to get an idea of what is essential and what can be reused.